spec/factories/installment_purchase.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  # Defining ramdomicaly an installment_quantity
+  rand_installments_quantity = rand(3..36)
+
+  # Defining ramdomicaly an total_amount
+  rand_total_amount = rand(100..100_000)
+
+  factory :installment_purchase do
+    description { Faker::Company.name }
+    total_amount { rand(100..100_000) }
```

Comment:
We need to have the same value here and at the `installment_amount` math:

```suggestion
    total_amount { rand_total_amount }
```